doc/themes/scikit-learn-modern/layout.html

```diff
@@ -21,8 +21,8 @@
   {% endblock %}
   <link rel="canonical" href="http://scikit-learn.org/stable/{{pagename}}.html" />
 
-  {% if favicon %}
-  <link rel="shortcut icon" href="{{ pathto('_static/' + favicon, 1) }}"/>
+  {% if favicon_url %}
+  <link rel="shortcut icon" href="{{ favicon_url|e }}"/>
```

Comment:
The `|e` is for [escaping HTML](https://jinja.palletsprojects.com/en/3.0.x/templates/#working-with-manual-escaping). It's a little safer to escape the string here.

Reply:
I thought that things are automatically escaped (instead of requiring template authors to remember to add `|e`)?